app/models/patient.rb

```diff
@@ -186,6 +190,10 @@ def save_identifier
     self.identifier = "#{line[0]}#{primary_phone[-5]}-#{primary_phone[-4..-1]}"
   end
 
+  def has_alt_contact?
+    !!(other_contact || other_phone || other_contact_relationship)
```

Comment:
Yeah, I had a similar hesitation because the `!!` really confused me when I first encountered it myself, and it's a tough thing to search for, to boot :)  From my perspective using `blank?` had readability issues of its own, but I just found `present?` which I think will do the trick!